docs/partner-twilio-mobile-services-how-to-use-voice-sms.md

```diff
@@ -219,23 +222,17 @@ Now that you've learned the basics of the Twilio service, follow these links to
 * [Twilio on GitHub] [twilio_on_github]
 * [Talk to Twilio Support] [twilio_support]
 
-[twilio_api_documentation]: http://www.twilio.com/api
 [twilio_security_guidelines]: http://www.twilio.com/docs/security
 [twilio_howtos]: http://www.twilio.com/docs/howto
 [twilio_on_github]: https://github.com/twilio
 [twilio_support]: http://www.twilio.com/help/contact
 [twilio_quickstarts]: http://www.twilio.com/docs/quickstart
 
-
-
-
 [twilio_rest_making_calls]: http://www.twilio.com/docs/api/rest/making-calls
 
 [twilio_pricing]: http://www.twilio.com/pricing
 [special_offer]: http://ahoy.twilio.com/azure
-[twilio_libraries]: https://www.twilio.com/docs/libraries
 [twiml]: http://www.twilio.com/docs/api/twiml
 [twilio_api]: http://www.twilio.com/api
```

Comment:
http://www.twilio.com/api is now https://www.twilio.com/docs/ . The is a redirection but it's probably better to point at the right thing.